# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../libvolume_id/makefile.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./extras/volume_id/lib/Makefile.orig	2006-11-25 15:28:14.000000000 -0300
+++ ./extras/volume_id/lib/Makefile	2006-11-25 15:28:33.000000000 -0300
@@ -17,6 +17,9 @@
 SHLIB_AGE = 0
 SHLIB = libvolume_id.so.$(SHLIB_CUR).$(SHLIB_REV).$(SHLIB_AGE)
 
+E=echo
+Q=@
+
 OBJS=				\
 	ext.o			\
 	fat.o			\
